### ISSUE TITLE
ISSUE: Multiple locks - code_slot entities becoming "unavailable" after adding second lock (v0.0.33g)

### ISSUE THREAD

**Issue by w0lf3h** [FutureTense/lock-manager#45]
Previous issue of simply re-writing entities has been fixed.

Currently, new code_slot_* entities are being created when adding multiple locks, however the sensors/entities of previous locks are becoming "unavailable."

For example:
I have frontdoor and backdoor locks. Updating backdoor results in creation of `sensor.backdoor_code_slot_*`.
Then, when I go to update frontdoor, `sensor.frontdoor_code_slot_*` is created, and status of `sensor.backdoor_code_slot_*` becomes "unavailable" (listed as "restored" under entity ids).

List of entity ids:
![Clipboard01](https://user-images.githubusercontent.com/1549104/97498571-b078ea80-1929-11eb-8903-bce733039633.jpg)

Lovelace snippet:
![Clipboard02](https://user-images.githubusercontent.com/1549104/97498586-b79ff880-1929-11eb-98d5-b0e6e5046f02.jpg)

*edit: This is on OZW 😊

**Comment by firstof9** [comment]
Weird it shouldn't be doing that anymore, I'll look into it.

**Comment by firstof9** [comment]
Are you running 0.0.34?

**Comment by w0lf3h** [comment]
This was on 0.0.33g. Haven't updated to 0.0.34 yet, and won't be able to till tomorrow morning!

**Comment by w0lf3h** [comment]
Issue is still persistent in v0.0.34

**Comment by firstof9** [comment]
Remove the ones with the red icon and restart Home Assistant please.

**Comment by w0lf3h** [comment]
Finally figured this out. This was my personal issue, but I'll leave it up here in case someone else has the same problem.

I actually ended up completely removing the entire integration and all associated entities, automations, etc. and starting from scratch. When I installed everything again, I realized that I had been using a single integration and renaming and customizing the options for all my different locks, which worked previously when all the sensors and synchronizing didn't exist, but *does not* work with any update past I believe 0.0.29. After making sure I had *separate* lock-manager integrations in my panel, my sensors were no longer disabling and overwriting one other on package generation. 👍
